tests/coverage

```diff
@@ -1 +1 @@
-86.9
+85.6
```

Comment:
This commit is also not signed. I assumed it was created with GitHub?

Reply:
This one is removed.